### DISCUSSION TITLE
Explain the dUNL on xrpl.org

### DISCUSSION THREAD

**Discussion by wojake** [XRPLBounties/Proposals#36] [Ideas]
Create a new section regarding dUNL: Explain what the dUNL is on xrpl.org, under [consensus network](https://xrpl.org/consensus-network.html).

Milestones: 2
Proposed Total Award: $3,000

| # | Description | Details | Proposed Award |
| --- | --- | --- | --- |
| 1 | Explain the basics | Describe what it is, who maintains it, who gets to be on it | $1,000 |
| 2 | Go in-depth | Explain in-depth details regarding dUNL amendment governance, network overlap, its say (dUNL) on ledger progress  | $2,000 |

**Comment by wojake** [comment]
Just confirming, Ripple will be the one to pay the recipient right?

**Reply by intelliot** [reply]
It depends. Ripple does approve and fund some bounties. We are also open to having other organizations and people fund bounties, if they choose.

**Comment by rippleitinnz** [comment]
A dUNL is simply a curated list that is already defined on xrpl.org and in other documents. A dUNL is not involved with amendment governance, network overlap, or ledger progress, It is simply a curated list. 

dUNL IMO is a name that infers something that it is not and should be changed to be more reflective of what it actually is.

I have spent weeks discussing this very thing, educating people on the difference. 

The UNL supergroup that is the 'defined' group of validators that the XRPL uses for 'governance' should be better defined by way of identity and all discussion should revolve around this, including such things as how much overlap is required to ensure a validator does not fork. 

![image](https://user-images.githubusercontent.com/42262747/188289119-92a40158-237f-450c-b5ff-250ae23a3650.png)

**Reply by xVet** [reply]
I hope it gets approved, it seems we definitely need a proper explanation for it 👏

**Reply by wojake** [reply]
To my knowledge, the dUNL has a say in ledger progress & amendment governance since it's the biggest overlap on the network. 

Yes, the dUNL is simply a curated list but you're understating what it really is and its influence since most validators use it for consensus (which increases its influence and say on ledger progress/amendment governance as their validation is taken into account by many validators across the network). Most validators simply use it because:
1. The dUNL consists of reliable, secure, trustworthy validators.
2. The dUNL is the **default** UNL when installing rippled, if config is untouched by the operator.
3. The dUNL is a reliable source of consensus since its validators are of high quality (refer *1)

It is a curated list of validators used for consensus, which is being used by many network participants which has a say in ledger progress and amendment governance. Their validation matters most on the network since many validators listen to their validation (biggest overlap). You're understating it by explaining what it is at its core concept (a curated list). While true, there's just more to it since it's being used for consensus.

But do correct me if I'm wrong, this is what I currently understand.

**Reply by wojake** [reply]
See why I want a well documented, reviewed, fact checked writing about the dUNL? I don't even know if I'm right!?

**Reply by rippleitinnz** [reply]
The only time the word default is used with regard the Validators UNL in Rippled is when it is describing those publishers who offer a list. in all other instances it simply refers to validators UNL's

There is no such thing as a 'default' UNL considered by the protocol, from my understanding, it is a complex calculation based on each validators UNL, determined by the protocol and not any centralized list (that could be the same as the list)

While Rippled ships with a config that uses a published list rather than the other options it doesn't mean the XRPL has a dUNL

It appears the term dUNL - described on xrpl.org as '...a default set from a publisher' is a poor term given it infers it has relevance to the Protocol when in fact, it doesn't.

![image](https://user-images.githubusercontent.com/42262747/188297999-17b675ba-6bc3-43f6-a1a6-5164d6eebc7b.png)
